src/CMakeLists.txt

```diff
@@ -366,11 +366,17 @@ if(USE_3DMAPPER)
   list(APPEND mudlet_HDRS glwidget.h)
 endif(USE_3DMAPPER)
 
+# credit to https://github.com/heavyai/heavydb/blob/72c90bc290b79dd30240da41c103a00720f6b050/CMakeLists.txt#L119
 macro(set_alternate_linker linker)
   find_program(LINKER_EXECUTABLE ld.${USE_ALTERNATE_LINKER} ${USE_ALTERNATE_LINKER})
   if(LINKER_EXECUTABLE)
     if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang" AND "${CMAKE_CXX_COMPILER_VERSION}" VERSION_LESS 12.0.0)
```

Comment:
According to https://github.com/rui314/mold#how-to-use the version check for the compiler (`clang`) is not helpful here as `-fuse-ld=mold` is what is always needed for that compiler.

😕

Reply:
@SlySven thanks for the catch! Fixed.